### ISSUE TITLE
LogTo does not work on HPC-GAP

### ISSUE THREAD

**Issue by mth162** [gap-system/gap#5665]
Build HPC-GAP:

./configure --enable-hpcgap
make
./bin/gap.sh
LogTo("/path/to/logfile")

then enter various GAP statements ...

### Observed behaviour
/path/to/logfile is created but interaction not captured (unless if there is an error--e.g. syntax error)

### Expected behaviour
Normal interaction show be captured.

### Copy and paste GAP banner (to tell us about your setup)

 ┌───────┐   HPC-GAP 4.12.2 built on 2023-11-07 21:34:51+0000
 │  GAP  │   https://www.gap-system.org
 └───────┘   Architecture: x86_64-pc-linux-gnu-hpcgap64-kv8
             Maximum concurrent threads: 4
 Configuration:  gmp 6.2.1, Boehm GC

#W <<< This is an alpha release.      >>>
#W <<< Do not use for important work. >>>

 Loading the library and packages ...
 Packages:   Alnuth 3.2.1, AutPGrp 1.11, CRISP 1.4.6, FactInt 1.6.3, FGA 1.4.0, GAPDoc 1.6.6, IRREDSOL 1.4.4, LAGUNA 3.9.5, Polycyclic 2.16, PrimGrp 3.4.4, SmallGrp 1.5.3, Sophus 1.27, TransGrp 3.6.4
 Try '??help' for help. See also '?copyright', '?cite' and '?authors'
gap>

**Comment by fingolfin** [comment]
Yes, this is a side effect of how the REPL is implemented in HPC-GAP. I don't think there is an easy way to fix this. I see little chance that anyone will work on this, I am afraid, sorry.
